zimui/src/components/video/VideoGrid.vue

```diff
@@ -9,23 +10,45 @@ const props = defineProps<{
   videos: VideoPreview[]
   playlistSlug?: string
 }>()
+
+const items = computed(() => props.videos.slice(0, 48))
+
+const loadMoreItems = async () => {
+  return new Promise<VideoPreview[]>((resolve) => {
+    setTimeout(() => {
+      resolve(props.videos.slice(items.value.length, items.value.length + 12))
+    }, 100)
+  })
+}
+
+const load = async ({ done }: { done: (status: 'ok' | 'empty') => void }) => {
+  const res = await loadMoreItems()
```

Comment:
Same remark about res

Reply:
Changed it in [d1930a0](https://github.com/openzim/youtube/pull/289/commits/d1930a05040e8a51b581c2be289d366f87a93806)